pkg/apis/storage/types.go

```diff
@@ -412,6 +412,15 @@ type CSIDriverSpec struct {
 	// +featureGate=SELinuxMountReadWriteOncePod
```

Comment:
Comment for CSINode object should also be updated to say it's mutable with the feature flag.

Reply:
Good idea, updated comment.